### ISSUE TITLE
Find a good replacement for ctf7

### ISSUE THREAD

**Issue by Sonix-** [teeworlds/teeworlds-maps#4]
I'd suggest to find a better map to replace the actual ctf7 since it's not that nice to keep 2 nearly identical maps (ctf1 - ctf7).
So I think that we should find/create a totally different and unique 2on2 map to replace it.

**Comment by ObaniGemini** [comment]
I think ctf7 should be enhanced insteed of being removed. I'm not a good mapper but as it's basic work I ca try to make a POC of this.
Let's do this !

**Comment by ObaniGemini** [comment]
I made this quick rework. If you want to enhance it... http://www.solidfiles.com/d/890aa27c01/

**Comment by heinrich5991** [comment]
Could you post a screenshot of your map?

**Comment by ObaniGemini** [comment]
This video reviews quickly the map and the problem I have on it (texture problem) : https://imgrush.com/qpla-grV5gs_

Please excuse the lags :)

**Comment by Sonix-** [comment]
I gave it a quick look and realized that you actually don't have much experience in making quality vanilla maps (no offense intended), and by quality I mean that the map has to have a great gameplay structure (and ofc a good design).
 You should first read this guide https://www.teeworlds.com/forum/viewtopic.php?id=6140 and then try to fix the map, some hints I can give you is to avoid creating narrow spaces they are just messy and not good, the Armour and the Hearts look spammed up and too many, and also avoid the use of death tiles in small maps (the ones next to the lasers, the pit is ok to use into some maps). :)

The texture problem is caused by a +1 tile blank space Map width in the Game layer

**Comment by ObaniGemini** [comment]
@Sonix- : this is my first map

**Comment by Sonix-** [comment]
@ObaniGemini : yeah I know, I'd suggest you first to start creating your own maps from scratch (and ofc post them on teeworlds forum cause I'll want to see them :) ) so you gain more and more experience at mapping :)

**Comment by hydroid7** [comment]
Last time when I played TW, i played on a custom Map named ctf_krypton. We're playing this on 2on2 gctf. But I think this would be a great replacement for ctf7. 

However, the map doesn't contain any Pickups and weapons, because it's a gctf Map. But we don't need big modificatons to make it Vanilla compatible. It also have a nice look.

I cannot present a link because I am not at home and im writing with my cell phone. ^^
->give me please 12 h for that.

**Comment by hydroid7** [comment]
So I forked this repo and uploaded the map. 
[ctf_krypton](https://github.com/hydroid7/teeworlds-maps)

**Comment by Dune-jr** [comment]
ctf7 was meant to replace ctf1 (because ctf1 is basically just dm1 rotated). But ctf1 was too historical to go away, I guess. Doesn't make much sense to scrap ctf7 tho imo
